gen/pb-rust/sigstore-protobuf-specs-codegen/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "sigstore-protobuf-specs-codegen"
+version = "0.0.1"
```

Comment:
(whoops comment didnt post) Does this version need to be in sync with the protobuf-specs version?

Reply:
Nope, this is purely an internal, helper-only crate (it performs the codegen for the actual crate, which will remain in sync with the protobuf-specs version).